### ISSUE TITLE
Nullable view columns exported as required 

### ISSUE THREAD

**Issue by louneskmt** [kristiandupont/kanel#421]
Hey!

I'm trying to export the type of the Postgres view of a base table, however the nullable columns are exported as required in the view type (but nullable as expected on the base table type). 

Here is an example:
```sql
CREATE TABLE test (
  this_is_nullable INT,
  this_is_not_nullable INT NOT NULL
);

CREATE VIEW test_view AS 
SELECT
  this_is_nullable,
  this_is_not_nullable
FROM test;
```
Outputs:
```ts
interface Test {
  this_is_nullable: number | null;
  this_is_not_nullable: number;
}

interface TestView {
  this_is_nullable: number;
  this_is_not_nullable: number;
}
```

I tested with a basic setup (no custom options) and with `resolveViews: true`, on `kanel v3.4.0`.

**Comment by kristiandupont** [comment]
Good catch, that's certainly a bug. I am not sure if it's in Kanel or in extract-pg-schema. I will try to look into it.

**Comment by kristiandupont** [comment]
I've published a version that hopefully fixes this. Please let me know if it worked!

**Comment by kristiandupont** [comment]
Right, I could have missed that. I'll look into it.

**Comment by louneskmt** [comment]
Just created a PR with the fix: #441.
